docs/dev/table/sql/show.md

```diff
@@ -91,6 +92,20 @@ tEnv.executeSql("SHOW TABLES").print();
 // |   my_table |
 // +------------+
 
+// create a table
+tEnv.executeSql("CREATE TABLE my_table (...) WITH (...)");
+// show create table my_table
+tEnv.executeSql("SHOW CREATE TABLE my_table").print();
+// +---------------------------+
+// |              create table |
+// +---------------------------+
+// | CREATE TABLE `my_table` ( |
+// |   ...                     |
+// | ) WITH (                  |
+// |   ...                     |
```

Comment:
shall it also prints sensitive options such as jdbc url contains password when on an external MySQL table?